oeps/best-practices/oep-0011/decisions/0004-js-ecma.rst

```diff
@@ -28,25 +28,27 @@ Rejected Alternatives
 
 .. _Reasons for rejecting CoffeeScript:
 
-1. **CoffeeScript**
+CoffeeScript
+============
 
-   The Open edX codebase made heavy use of CoffeeScript for several years,
-   but its use at edX has now been officially deprecated. Most of the language
-   benefits of CoffeeScript were made available in JavaScript as of the ES2015 spec,
-   and CoffeeScript's community has largely moved on.
+The Open edX codebase made heavy use of CoffeeScript for several years,
+but its use at edX has now been officially deprecated. Most of the language
+benefits of CoffeeScript were made available in JavaScript as of the ES2015 spec,
+and CoffeeScript's community has largely moved on.
 
 .. _Reasons for rejecting TypeScript:
 
-2. **TypeScript**
-
-   Several variants of typed JavaScript have grown in community popularity
-   recently, with `TypeScript`_ as the most popular. edX believes adding
-   typing to our JavaScript would raise the barrier to entry for less
-   experienced front end developers, and that using TypeScript in particular
-   would lock us into a tool with an uncertain future (as we experienced with
-   CoffeeScript). If typing is an absolute necessity for a feature
-   (e.g., an API client or sensitive business logic), `Flow`_ is recommended,
-   as it can be incrementally added and evaluated alongside non-typed code.
+TypeScript
+==========
+
+Several variants of typed JavaScript have grown in community popularity
+recently, with `TypeScript`_ as the most popular. edX believes adding
```

Comment:
```suggestion
recently, with `TypeScript`_ as the most popular. The Open edX community believes adding
```

But really, should this section be rewritten since you're adding an ADR about actually using TypeScript? It's ok to update an OEP, just the update should be recorded in the change history at the bottom

Reply:
Sorry, I thought this was the OEP file not an ADR file. I would make the suggested language change but keep the updated TypeScript recommendation in ADR 7.